rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
 profile = "default"
-channel = "1.66.0"
+channel = "1.67.1"
```

Comment:
Whatever fix is required for this update probably deserves its own PR (doesn't appear to be working in this draft yet)

Reply:
Reverted